### ISSUE TITLE
Uncaught TypeError: document.getElementById(...) is null

### ISSUE THREAD

**Issue by Garinmckayl** [boopathikumar018/docsify-darklight-theme#14]
the plugin is working flawlessly most of the time, but once in a every couple of times the plugin is messing the other plugins that i am using like tab. so they will stop working completely. but this plugin works fine

**Comment by edufschmidt** [comment]
I'm observing the same behavior. Have you managed to find a workaround?

**Comment by edufschmidt** [comment]
Update: adding a `null` check on line 112 of the file [docs/js/index.js](https://github.com/boopathikumar018/docsify-darklight-theme/blob/master/docs/js/index.js) does the trick:

```js
const el = document.getElementById('docsify-darklight-theme')

if (el !== null) {
    el.addEventListener('click', function() { ... })
}
```

**Comment by Garinmckayl** [comment]
yeah, that does the trick

**Comment by boopathikumar018** [comment]
Added the fix in version: 3.1.4
